Detections/DeviceEvents/SolardWinds_TEARDROP_Process-IOCs.yaml

```diff
@@ -0,0 +1,47 @@
+id: 738702fd-0a66-42c7-8586-e30f0583f8fe
+name: TEARDROP memory-only dropper
+description: |
+  Identifies SolarWinds TEARDROP memory-only dropper IOCs in Window’s defender Exploit Guard activity
+  References:
+  - https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html
+  - https://gist.github.com/olafhartong/71ffdd4cab4b6acd5cbcd1a0691ff82f
+severity: High
+requiredDataConnectors:
+  - connectorId: Microsoft 365 Defender
+    dataTypes:
+      - DeviceEvents
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+  - Persistence
+relevantTechniques:
+  - T1012
+  - T1027
+  - T1543.003
+  - T1195.002
+query:  |
+
+  DeviceEvents
+  | where ActionType contains "ExploitGuardNonMicrosoftSignedBlocked"
+  | where InitiatingProcessFileName contains "svchost.exe" and FileName contains "NetSetupSvc.dll"
```

Comment:
For performance, lets change these from 'contains' to 'has'

Reply:
Will do so in another PR, it could probably just be a `==` too.